### DISCUSSION TITLE
Generating XIC profile

### DISCUSSION THREAD

**Discussion by alexproteomics** [vdemichev/DiaNN#378] [General]
Hi Vadim,
I was wondering if there is a way to generate the XIC of the fragments quantified for specific features in each run. I found the columns Fragment.Quant.Raw and Fragment.Info (I set --report-lib-info). I'm guessing the Fragment Quant here is actually the summed intensities of each fragment in Fragment.Info across time. Thus I want to go one step before this summarization and get the information of each fragment across time. I suspect this information is in the .quant file but I don't know how to read this. BTW, I'm using Any LC setting. 
In addition, I would like to get further information of each feature (fragment) such as FWHM, I know the pdf report has the mean of all features, is there a way to get the information from each feature?
As always, I really appreciate your work. 
thanks
Alex

**Comment by vdemichev** [comment]
Hi Alex,

Yes, please see the --vis command for this.
Info like FWHM is currently not reported for individual precursors, although RT.Stop - RT.Start difference is roughly proportional to the peak width, in the Any LC (high accuracy) mode.

Best,
Vadim

**Reply by alexproteomics** [reply]
Thanks Vadim. I had missed this command. I will try it.
cheers
Alex

**Comment by cps36** [comment]
Hi @vdemichev , 

Thanks for the hint about the --vis command. How exactly does the "N" parameter work? It is not so clear to me what precisely it is doing. 

Best,

Chris

**Reply by vdemichev** [reply]
Thanks for spotting, I corrected the description

**Reply by cps36** [reply]
Awesome - thanks a lot!
